### DISCUSSION TITLE
Getting error after redirection from Google Auth

### DISCUSSION THREAD

**Discussion by kdddion** [Budibase/budibase#8325] [Help]
Hi there,

Trying to set up Google authentication on my company's self-hosted budibase docker container. After getting the tokens, setting a proper redirect url, etc. Everything works fine until the moment I get redirected, where I get this:

message | "Cannot read property 'provider' of undefined"
status | 500

Could I have some assistance as to what may be causing this?

I should note that I am using a custom redirect url in the form of budibase.(domain) and I have added entries to our dns servers to be able to resolve that. Everything's using http and is not accessible from anywhere else than locally

**Comment by melohagan** [comment]
Hey @kdddion 

Could you provide the full error message, including any stack traces please?
